include/swift/AST/DiagnosticsSema.def

```diff
@@ -1122,6 +1122,9 @@ ERROR(invalid_dynamic_member_lookup_type,none,
       "@dynamicMemberLookup attribute requires %0 to have a "
       "'subscript(dynamicMember:)' method that accepts either "
       "'ExpressibleByStringLiteral' or a keypath", (Type))
+ERROR(invalid_dynamic_member_subscript, none,
```

Comment:
I think this should be a `NOTE` to complement an error about `@dynamicMemberLookup` requirements?

Reply:
Oops, you're right! Done.